### DISCUSSION TITLE
关于本地搭建好之后的配置

### DISCUSSION THREAD

**Discussion by lobmoo** [Qexo/Qexo#472] [Q&A] [answered]
![image](https://github.com/Qexo/Qexo/assets/45560941/c6ccb5cf-dc26-4985-a896-f64efdf29354)
这里，我填写的本地地址没问题，，请问github地址是干啥的，，真的虚心求教，我不填github地址强制提交报错

**Comment by miaowmint** [comment, marked as answer]
把服务商改成本地
